CMakeLists.txt

```diff
@@ -93,11 +93,11 @@ install(EXPORT umpire-targets DESTINATION share/umpire/cmake)
 add_subdirectory(src)
 
 if (ENABLE_TESTS)
-  set (UMPIRE_TEST_OUTPUT_DIR ${CMAKE_BINARY_DIR}/test)
+  set (UMPIRE_TEST_OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY})
```

Comment:
Is this a CMake variable?

Reply:
It seems to be something defined by BLT.
```
~/Umpire/blt $ git grep TEST_OUTPUT_DIRECTORY
SetupBLT.cmake:    set(TEST_OUTPUT_DIRECTORY
```

Reply:
Perfect!